src/Analysis/Engine/Impl/Values/BuiltinFunctionInfo.cs

```diff
@@ -93,7 +93,7 @@ public override bool IsOfType(IAnalysisSet klass)
                 return name;
             }
             name = type.Name + "." + name;
-            if (type.IsBuiltin || type.DeclaringModule == null) {
+            if (type.IsBuiltIn || type.DeclaringModule == null) {
```

Comment:
Kinda inconsistent. It is still `Builtin` in other places.

Reply:
OK, all to `IsBuiltin` then